src/comps/product-listing/productListing.jsx

```diff
@@ -0,0 +1,73 @@
+import "./productListing.css"
+
+
+import { useProduct} from "./../../context/products.context.jsx"
+
+
+const ProductListing = ()  => {
+
+    const {itemList, setItemList} = useProduct()
+
+    return (
+
+ 
+    <main className="product-display-container">
+
+
+        <div className="product-display-component">
+            <div className="product-grid">
+             
+
+              
+
+                { itemList.map( ({title , seller ,price,categoryName,image,rating}) => 
+                { 
+                    return(
+                        <div className="ecomm vertical-card">
```

Comment:
you can move the entire code inside the return statement and make it a separate component, this way the code will look clean and could be reused in future